modules/Bio/EnsEMBL/Analysis/Hive/Config/RepeatMasking.pm

```diff
@@ -0,0 +1,549 @@
+                         dust_path => $self->o('dust_path'),
+                       },
+        -rc_name    => 'simple_features',
+        -flow_into => {
+                         1 => ['run_trf'],
+                         -1 => ['run_trf'],
+		         -2 => ['run_trf'],
+                      },
+        -hive_capacity => $self->hive_capacity_classes->{'hc_high'},
+        -batch_size => 20,
+      },
+
+
+      {
+        # Run TRF
+        -logic_name => 'run_trf',
+        -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveAssemblyLoading::HiveTRF',
+        -parameters => {
+                         target_db => $self->o('reference_db'),
+                         logic_name => 'trf',
+                         module => 'HiveTRF',
+                         trf_path => $self->o('trf_path'),
+                       },
+        -rc_name    => 'simple_features',
+        -hive_capacity => $self->hive_capacity_classes->{'hc_high'},
+        -batch_size => 20,
+      },
+
+    ];
+}
+
+
+sub resource_classes {
+  my $self = shift;
+
+  return {
+    '2GB' => { LSF => $self->lsf_resource_builder('production-rh74', 2000, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    '3GB' => { LSF => $self->lsf_resource_builder('production-rh74', 3000, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    '5GB' => { LSF => $self->lsf_resource_builder('production-rh74', 5000, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    'default' => { LSF => $self->lsf_resource_builder('production-rh74', 900, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    'repeatmasker' => { LSF => $self->lsf_resource_builder('production-rh74', 2900, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    'repeatmasker_rebatch' => { LSF => $self->lsf_resource_builder('production-rh74', 5900, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+    'simple_features' => { LSF => $self->lsf_resource_builder('production-rh74', 2900, [$self->default_options->{'pipe_db_server'}, $self->default_options->{'dna_db_server'}], [$self->default_options->{'num_tokens'}])},
+  }
+}
+
+sub hive_capacity_classes {
+  my $self = shift;
+
+  return {
+           'hc_very_low'    => 35,
+           'hc_low'    => 200,
+           'hc_medium' => 500,
+           'hc_high'   => 1000,
+         };
+}
+
+
+sub check_file_in_ensembl {
+  my ($self, $file_path) = @_;
+  push @{$self->{'_ensembl_file_paths'}}, $file_path;
+  return $self->o('enscode_root_dir').'/'.$file_path;
+}
```

Comment:
This is not used anywhere, it can be removed

Reply:
Change effected